spec/tags/sms.md

```diff
@@ -20,4 +27,6 @@ When you receive a message from one contact, the sender and recipient is inverte
 * Recipient: is the SMS sender id
 * Sender: is the phone number of contact
 
-In API the sender is the field `from` and the receiver is the field `to` of message object.
\ No newline at end of file
+In API the sender is the field `from` and the receiver is the field `to` of message object.
```

Comment:
Já tava errado, mas em vez de "In API the sender is the field" seria algo tipo "In the API the sender is in the field"